cleaner-fix.sh

```diff
@@ -231,8 +263,14 @@ extract() {
     fi
     for f in $apps; do
         echo "----> copying $f..."
-        $sevenzip x -odeodex/system/ "$img" priv-app/$f >/dev/null || clean "$work_dir"
+        if [[ "$f" == "DeskClock" ]]; then
+            $sevenzip x -odeodex/system/ "$img" app/$f >/dev/null || clean "$work_dir"
+        else
+            $sevenzip x -odeodex/system/ "$img" priv-app/$f >/dev/null || clean "$work_dir"
+        fi
     done
+    mv deodex/system/app/DeskClock deodex/system/priv-app/DeskClock
+    rm -r deodex/system/app
```

Comment:
Because the other places are always dealing with apps in priv-app folder, so I also move DeskClock.app from app folder into priv-app folder. This would eventually leads to a side effect: duplicated apps in both app and priv-app folders after flashing the zip package, but it seems has no harm.